project.yaml

```diff
@@ -181,11 +181,13 @@ targets:
         test_distros:
           representative:
           - windows-cloud:windows-2016-core
-          - windows-cloud:windows-2022
+          - windows-cloud:windows-2025
           - windows-sql-cloud:sql-std-2019-win-2019
-          - windows-sql-cloud:sql-std-2022-win-2022  
+          - windows-sql-cloud:sql-std-2022-win-2022
```

Comment:
We should also demote this to `exhaustive` and add `sql-std-2022-win-2025` to `representative`

Reply:
Done in [44238a1](https://github.com/GoogleCloudPlatform/ops-agent/pull/1845/commits/44238a1c81d809f95ba9d0a1357c6126a394bddd)